### PR TITLE
Chapter 2/Homework 4

### PR DESCRIPTION
**Pull request by Tailor-AI**
First draft of homework 4.

### CODE REVIEW

File: Src/BootCamp.Chapter/GlobalSettings.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public static class GlobalSettings
+    {
+        public static string filePath = Directory.GetCurrentDirectory();
+
+        public static ILogger Logger(LogType logType)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/GlobalSettings.cs]: Methods should be verbs. What do you do with logger?
- get logger?
- create logger?
- load logger?
?

File: Src/BootCamp.Chapter/Logger/FileLogger.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class FileLogger : ILogger
+    {
+        public void LogMessage(DateTime dateTime, string message)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Logger/FileLogger.cs]: DateTime is not really needed.
It doesn't make sense to log any other time than DateTime.Now.
If you want date to sometimes be there, sometimes not, then just pass it use another method or configuration. I imagine that you either want dates always or never, never a mix.

**Review by Almantask — CHANGES_REQUESTED** [review]

File: Src/BootCamp.Chapter/GlobalSettings.cs (hunk 1 above)
Comment (Tailor-AI) [reply Src/BootCamp.Chapter/GlobalSettings.cs]: Correct, renamed this to CreateLog.

File: Src/BootCamp.Chapter/Logger/FileLogger.cs (hunk 2 above)
Comment (Tailor-AI) [reply Src/BootCamp.Chapter/Logger/FileLogger.cs]: Removed the parameter and put the DateTime.Now in the message itself.

**Review by Almantask — APPROVED** [review]

**Pull request closed without merging**